evansmusomi/burger-builder
# Fetches Ingredients from Server

*What this does*
- Loads ingredients from server
- Improves global error handling

*How to verify*
- User can see the ingredient state that's on firebase when they load the app
- User can see an error message if an error occurs while loading the ingredients
